internal/completions/completions.go

```diff
@@ -60,7 +60,7 @@ func (c Completer) AddInfo(targetInfoKey string, additionalFormatting ...string)
 			return Completer{}
 		}
 
-		info := fmt.Sprintf("%v", infoRaw)
+		info := fmt.Sprintf("%v ", infoRaw)
```

Comment:
Remove this space , it will be added to all completions, also it causes this pkg's tests to fail